recipe/meta.yaml

```diff
@@ -117,7 +117,7 @@ requirements:
     - nspr                               # [unix]
     - nss                                # [unix]
     - libsqlite
-    - mysql-devel                        # [not win]
+    - mysql-devel ==8.0                  # [not win]
```

Comment:
why is this pinned? can we unpin?

Reply:
it seems maybe there wasn't a global migrator.

this definitely needs one it seems: 

Trying to add one here
https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/5637

Reply:
hmm, not fun:

https://github.com/qt/qtbase/commit/7d99fa8e5bf036f97d8ef41780f6e401b32f47ef#diff-1d355c25c0b0eddec2be48253407780c4dc510d986739aec61e1ec892ccaf86e

seems like they changed the API: https://dev.mysql.com/doc/relnotes/mysql/8.3/en/news-8-3-0.html

Reply:
I tried backporting a commit that claimed to fix the build for mysql 8.3 but it didn't help. Figure it's better to update qt without updating mysql than not updating either.

Reply:
Do you mind giving https://github.com/conda-forge/qt-main-feedstock/pull/244 a quick review for the patch. I think I got it right. But as somebody who has already looked at the peticular code path your eyes are very valuable

Reply:
Sorry I don't really understand the changes and wouldn't know how to test it. That's why I gave up trying to patch it and resorted to pinning it.

Reply:
understood.